apps/client/src/utils/keycloak/keycloak.ts

```diff
@@ -27,6 +27,19 @@ let keycloak: Keycloak
 export function getKeycloak() {
   if (!keycloak) {
     keycloak = new Keycloak(keycloakConfig)
+    keycloak.onAuthSuccess = () => {
+      if (!(keycloak.refreshTokenParsed?.exp && keycloak.tokenParsed?.exp && keycloak.refreshTokenParsed.exp < keycloak.tokenParsed.exp)) {
+        return
+      }
+      console.warn('You keycloak seems having bad settings, refresh token expires before token')
```

Comment:
Qu'as-tu voulu dire en français ?

Reply:
ce warn ne devrait pas être dans le if au dessus du return ?

Reply:
en gros "le server keycloak est mal configuré car le refreshToken expires avant le token."

et non en l'occurrence le return c'est si c'est bien configuré. mais je continue

Reply:
`Keycloak misconfiguration : refreshToken should not expire before token.`